frontend/ui/src/pages/Channels/ChannelsMainPage/SourceInfo/index.tsx

```diff
@@ -59,9 +60,9 @@ const SourceInfo = (props: SourceInfoProps) => {
                   );
                 })}
               </div>
-              <div className={styles.extraChannel}>
+              <div className={styles.extraChannel} onClick={() => props.onSourceInfoClick()}>
                 {hasExtraChannels && (
-                  <LinkButton onClick={props.onMoreChannelsClick}>
+                  <LinkButton onClick={() => props.onSourceInfoClick()}>
```

Comment:
shouldn't we bind in the constructor? This is not recommended by the react team (as it puts additional load on the runtime by creating a function _every time_ the component renders)

Probably also a more general point (as I've seen this quite a bit in the codebase already I think) /cc @AitorAlgorta

Reply:
It's true that we miss-used this feature in some places, but I think here it makes sense. We don't want to use the HTML event that we get in the `onClick` we just want to trigger a function when the `onClick` gets triggered, if we would do it differently we would have to pass back the `event` that the `onClick` give us and we won't use it. That's how I see it here @lucapette